### ISSUE TITLE
SORBS went out of business

### ISSUE THREAD

**Issue by cbrandlehner** [E-F-A/v4#371]
ORBS (Spam and Open Relay Blocking System) has gone out of business and is no longer available as a service. As a result, it should be removed from all sources within E-F-A to ensure that the system remains efficient and up-to-date.

Information source: [https://www.heise.de/news/Ausgeblockt-Antispam-Blockliste-SORBS-ist-abgeschaltet-9752366.html](https://www.heise.de/news/Ausgeblockt-Antispam-Blockliste-SORBS-ist-abgeschaltet-9752366.html)

Steps to Reproduce:

Check current sources for spam filtering within E-F-A.
Identify that SORBS is still listed as an active source.
Expected Behavior:

SORBS should be removed from all source lists to avoid reliance on an unavailable service.

Actual Behavior:

SORBS is still listed as an active source within E-F-A.

Possible Solution:

Locate all configurations and scripts where SORBS is referenced.
Remove SORBS from these configurations and scripts.
Test to ensure that the removal does not impact other functionalities.

Additional Context:

SORBS was once a valuable resource for blocking spam and open relays, but with it no longer being operational, continuing to reference it may lead to inefficiencies or errors in spam detection.

Thank you for addressing this issue to maintain the integrity of our spam filtering system.

**Comment by shawniverson** [comment]
@cbrandlehner Where are you seeing SORBS in eFa? The SpamAssassin project just pushed out rule updates that remove SORBS rulesets, so an automatic or manual sa-update should remove all references to SORBS in the default installation.

**Comment by cbrandlehner** [comment]
I am seeing this in received emails (235 hits last month):
RCVD_IN_SORBS_WEB	SORBS: sender is an abusable web server

and found SORBS configured in etc/MailScanner/spam.lists.conf

**Comment by shawniverson** [comment]
Yeah spam.lists.conf isn't being used unless you have "Spam Lists" defined in MailScanner.conf.  RCVD_IN_SORBS_WEB is a spamassassin rule.  You should stop seeing these if you haven't already since they have been removed from the ruleset.
